### DISCUSSION TITLE
🚀 Mac mini with M2 chip will be default starting September 1, prices unchanged

### DISCUSSION THREAD

**Discussion by mikhail-tokarev** [codemagic-ci-cd/codemagic-docs#2659] [Announcements]
We are upgrading our infrastructure and replacing all Mac mini M1 machines with newer and more powerful M2 machines on all plans – with no price change. This allows us to offer our users faster builds at the same price. Check out the M2 machine specs in our[ documentation](https://docs.codemagic.io/specs/versions-macos/).

The transition from M1 machines to M2 machines will occur on **September 1, 2024**.
Here’s what you should know about this change.

# Migration is automatic
The migration will be automatic and no immediate changes are required (however, please see the next section if your repository is behind a firewall). We don’t expect this to bring along downtime or breaking changes as the M2 machines have the same macOS version and preinstalled software as M1 machines. 

Starting September 1, all builds configured to run on `mac_mini_m1` will use `mac_mini_m2` instead. While the existing configuration will continue to work, we advise you to update the `mac_mini_m1` instance type to `mac_mini_m2` in codemagic.yaml. Otherwise, you’ll be seeing a message about using a deprecated instance type in build logs.  

# New IP addresses to whitelist
As we are growing, we are adding a new data center to provide more machines to our users. If your repository is behind a firewall, you will need to update the list of IP addresses allowed to access the repository. See our [documentation](https://docs.codemagic.io/getting-started/adding-apps/#firewall-configuration-for-privately-hosted-repositories) for the full list of IP address ranges our build machines use.

# No change in prices
Mac mini with M2 will replace M1 on all plans with no changes to plan prices.
- The **Free tier** will continue to include 500 free build minutes but builds will run on Mac mini M2 hardware. No multiplier applied. Extra minutes are available for $0.095/minute.
- The **Pay-as-you-go** price for Mac mini with M2 is $0.095/minute, which is the same as the current price for Mac mini M1.
- The **Fixed-price plan** includes 3 concurrencies and unlimited build minutes on Mac mini with M2 for $399/month ($3,990/year if paid annually). Machines with M2 Pro, M2 Max and M2 Ultra chips are available on request, please contact us for a trial.

As always, don’t hesitate to get in touch with us if you have any questions or concerns.

**Comment by helinanever** [comment]
We’ve completed the migration and all builds, including builds on the free plan, now run on Mac mini M2 by default. :tada:

Let us know if you notice any improvements in build speed after switching from M1 to M2 by tagging [@codemagicio](https://x.com/codemagicio) on X.
